travis/cfg/travis_run_pylint_pr.cfg

```diff
@@ -42,6 +43,7 @@ enable=api-one-multi-together,
     no-utf8-coding-comment,
     openerp-exception-warning,
     rst-syntax-error,
+    too-complex,
```

Comment:
Currently all pr.cfg messages are really beta mssages:
https://github.com/OCA/maintainer-quality-tools/issues/254
(Without affect travis status build).

Then too-complex checks don't affect travis build status too.

If we use the proposal process, I will create a new pr to add beta.cfg, where we will add too-complex.